py/desispec/scripts/qa_prod.py

```diff
@@ -6,7 +6,7 @@
 
 
 def parse(options=None):
-    parser = argparse.ArgumentParser(description="Generate/Analyze Production Level QA [v1.3]")
+    parser = argparse.ArgumentParser(description="Generate/Analyze Production Level QA [v1.4.1]")
```

Comment:
Is QA versioning separate from the desispec versioning needed?  Perhaps, but let's be explicit about that choice and reasons.

Maintaining this version string is going to be a pain, e.g. elsewhere in this PR you refer to version 1.4 instead of 1.4.1...  Let's add a QA version variable, and populate the usage/help messages from that so that updating the version number only requires updating one place.